src/runtime-rs/crates/resource/src/rootfs/share_fs_rootfs.rs

```diff
@@ -23,14 +23,21 @@ impl ShareFsRootfs {
         share_fs_mount: &Arc<dyn ShareFsMount>,
         cid: &str,
         bundle_path: &str,
-        rootfs: &Mount,
+        rootfs: Option<&Mount>,
+        is_mounted: bool,
     ) -> Result<Self> {
-        let bundle_rootfs = format!("{}/{}", bundle_path, ROOTFS);
-        rootfs.mount(&bundle_rootfs).context(format!(
-            "mount rootfs from {:?} to {}",
-            &rootfs, &bundle_rootfs
-        ))?;
-
+        let bundle_rootfs = if is_mounted {
```

Comment:
Is `is_mounted == false` == `rootfs.is_some()`? so you can save the `is_mounted` parameter.

Reply:
Thanks, fixed